modules/navigation/nav_map.cpp

```diff
@@ -54,6 +54,10 @@
 		r_path_owners->push_back(poly->owner->get_owner_id()); \
 	}
 
+#ifdef DEBUG_ENABLED
+const String NAVMAP_ITERATION_ZERO_ERROR_MSG = "NavigationServer navigation map query failed because it was made before first map synchronization.\nNavigationServer 'map_changed' signal can be connected to receive update notifications.";
```

Comment:
1) consider using macro instead of constant
2) consider changing `connected` to `used` in the string itself